### DISCUSSION TITLE
关于S905-B芯片的问题

### DISCUSSION THREAD

**Discussion by Windtweet** [ophub/amlogic-s9xxx-armbian#1521] [Amlogic]
请问S905-B芯片是不是不在您这里的支持列表中？
我自己查找后发现 https://www.right.com.cn/forum/thread-8221485-1-1.html 中提到了“TTL线进uboot里改env参数”的方法，但我在网上并未查到有关信息。
我不希望麻烦您对此芯片加入支持，但我想请教一下，如果我要自己在这个芯片上跑Armbian，有什么可以参考的资料吗？
耽误您时间了，谢谢！

**Comment by ophub** [comment]
可以看一下文档里面的分区确认的方法，把armbian写入安全区即可
